src/shims/unix/sync.rs

```diff
@@ -206,7 +206,7 @@ fn translate_kind<'tcx>(ecx: &MiriInterpCx<'tcx>, kind: i32) -> InterpResult<'tc
 // - id: u32
 
 #[derive(Debug)]
-/// Additional data that may be used by shim implementations.
+/// Additional data that we attach with each rwlock instance.
```

Comment:
I also changed this documentation comment since `AdditionalRwLockData` is now something internal to the shim implementation, so the previous wording felt a bit off.